src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
@shaurya947 I was using the `context` props before. I completely overlook this during the rebase! Is `muiTheme` working?